docs/source/lsp.md

```diff
@@ -0,0 +1,5 @@
+---
+title: Lobster LSP
+---
+
+TODO
```

Comment:
Yes, lets have some description on how to install it and use it :)

Reply:
Oops forgot that I left a TODO there 😅